src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/ScannableItemResponse.java

```diff
@@ -0,0 +1,131 @@
+
+    @JsonProperty("next_action")
+    private String nextAction;
+
+    @JsonSerialize(using = CustomTimestampSerialiser.class)
+    @JsonProperty("next_action_date")
+    private Timestamp nextActionDate;
+
+    @JsonProperty("ocr_data")
+    private String ocrData;
+
+    @JsonProperty("file_name")
+    private String fileName;
+
+    @JsonProperty("notes")
+    private String notes;
+
+    @JsonProperty("document_url")
+    private String documentUrl;
+
+    @JsonCreator
+    public ScannableItemResponse(
+        @JsonProperty("document_control_number") String documentControlNumber,
+        @JsonDeserialize(using = CustomTimestampDeserialiser.class)
+        @JsonProperty("scanning_date") Timestamp scanningDate,
+        @JsonProperty("ocr_accuracy") String ocrAccuracy,
+        @JsonProperty("manual_intervention") String manualIntervention,
+        @JsonProperty("next_action") String nextAction,
+        @JsonDeserialize(using = CustomTimestampDeserialiser.class)
+        @JsonProperty("next_action_date") Timestamp nextActionDate,
+        @JsonProperty("ocr_data") String ocrData,
+        @JsonProperty("file_name") String fileName,
+        @JsonProperty("notes") String notes
+    ) {
+        this.documentControlNumber = documentControlNumber;
+        this.scanningDate = scanningDate;
+        this.ocrAccuracy = ocrAccuracy;
+        this.manualIntervention = manualIntervention;
+        this.nextAction = nextAction;
+        this.nextActionDate = nextActionDate;
+        this.ocrData = ocrData;
+        this.fileName = fileName;
+        this.notes = notes;
+    }
+
+    public String getFileName() {
+        return fileName;
+    }
+
+    public String getDocumentUrl() {
+        return documentUrl;
+    }
+
+    public void setDocumentUrl(String documentUrl) {
+        this.documentUrl = documentUrl;
+    }
+
+    public String getDocumentControlNumber() {
+        return documentControlNumber;
+    }
+
+    public Timestamp getScanningDate() {
+        return scanningDate;
+    }
+
+    public String getOcrAccuracy() {
+        return ocrAccuracy;
+    }
+
+    public String getManualIntervention() {
+        return manualIntervention;
+    }
+
+    public String getNextAction() {
+        return nextAction;
+    }
+
+    public Timestamp getNextActionDate() {
+        return nextActionDate;
+    }
+
+    public String getOcrData() {
+        return ocrData;
+    }
+
+    public String getNotes() {
+        return notes;
+    }
+
+    @Override
+    public String toString() {
+        return "ScannableItemResponse{"
+            + "documentControlNumber='" + documentControlNumber + '\''
+            + ", scanningDate=" + scanningDate
+            + ", ocrAccuracy='" + ocrAccuracy + '\''
+            + ", manualIntervention='" + manualIntervention + '\''
+            + ", nextAction='" + nextAction + '\''
+            + ", nextActionDate=" + nextActionDate
+            + ", ocrData='" + ocrData + '\''
```

Comment:
ocrdata should not be outputted

Reply:
in toString()? why? is it potentially too big?

Reply:
it's user data so definitely we should not log it :)